pwiz_tools/Skyline/TestFunctional/IrtTest.cs

```diff
@@ -143,8 +143,8 @@ private void RunIrtTest()
                       {
                           Assert.AreEqual(numStandardPeps, irtDlg1.StandardPeptideCount);
                           //And that there are 3 below 0 and 3 above 100
-                          Assert.AreEqual(3, irtDlg1.StandardPeptides.Count(pep => pep.Irt < 0));
-                          Assert.AreEqual(3, irtDlg1.StandardPeptides.Count(pep => pep.Irt > 100));
+                          Assert.AreEqual(3, irtDlg1.StandardPeptides.Count(pep => Math.Round(pep.Irt, 2) < 0));

```

Comment:
Would it make more sense if this were:
Assert.AreEqual(3, irtDlg1.StandardPeptides.Count(pep => pep.Irt < -0.005));

Reply:
Maybe, I like the rounding though because it's comparing the numbers that are visible to the user in the dialog.